### DISCUSSION TITLE
Custom !include tag

### DISCUSSION THREAD

**Discussion by kalemas** [redhat-developer/vscode-yaml#575] [General]
Hello, i had implemented custom tag that adds tags from another file like this:
![image](https://user-images.githubusercontent.com/638393/129321023-15ea465d-40b6-4f84-b359-ff13504f2428.png)
and so as you can see it cant be handler by yaml validator. Just want to ask if it possible to add support for such. I m using PyYaml custom Loader.
Thank you.

**Comment by kalemas** [comment]
maybe it is possible to ignore some validation issues?

but it may be best to provide loader class if static analysis is related to python somehow.

**Reply by kalemas** [reply]
I was included all referenced anchors in top of the file, before `!include` custom tag, and i m able to pass document validation. Skipping undefined anchor issue or providing loader class to resolve anchors is still nice to have.

**Comment by FALLAI-Denis** [comment]
Hi,

We also have a need to establish references between YAML documents, which does not exist in the YAML standard.

Can you detail your solution?

Thank you.
